cromulent/vocab.py

```diff
@@ -138,8 +138,8 @@ def register_instance(name, data):
 	"Auctioneer":  {"parent": Person, "id":"300025208", "label": "Auctioneer"}, # is this useful?
 
 
-	"AuctionEvent": {"parent": Activity, "id":"300xxxxxx", "label": "Auction Event"},
-	"Auction":     {"parent": Activity, "id":"300054751", "label": "Auction of Lot"}, # Individual auction-of-lot
+	"AuctionEvent": {"parent": Activity, "id":"300054751", "label": "Auction Event"},
+	"Auction":     {"parent": Activity, "id":"300420001", "label": "Auction of Lot"}, # Individual auction-of-lot
```

Comment:
I don't see any corresponding data at http://vocab.getty.edu/aat/300420001 (?)

Reply:
It's in staging for release to the system in the next export.

Reply:
OK. Then this latest change since my last approval seems trivial.